Cilsil/Program.cs

```diff
@@ -100,7 +101,14 @@ private static void Main(string[] args)
 
             (var cfg, var tenv) = ExecuteTranslation(paths, extprogress);
 
-            File.WriteAllText(cfgtxt ?? "./cfg.txt", cfg.ToString());
+            var Utf8Encoder = Encoding.GetEncoding(
```

Comment:
Maybe use other dedicated placeholders, instead of `string.empty` or nothing, make more sense when encoding failed?

You may take a look at [this example](https://docs.microsoft.com/en-us/dotnet/api/system.text.encoderreplacementfallback?redirectedfrom=MSDN&view=net-6.0) for inspiration.

Reply:
I actually don't think we should add unnecessary characters. Every extra character adds unnecessary disk/IO which is not of value to the backend analysis -- what is important is that the fallbackexception is applied consistently.